src/PipParser.js

```diff
@@ -9,30 +9,30 @@ const DEFAULT_PIP_FILENAMES = {
     acpSearchProvider: 'acpSearchProvider.xml',
     acpTemplate: 'acptemplates.tar',
     bbcode: 'bbcode.xml',
-    box: null,
-    clipboard: null,
-    coreObject: null,
-    cronjob: null,
-    eventListener: null,
+    box: 'box.xml',
+    clipboard: 'clipboard.xml',
+    coreObject: 'coreObject.xml',
+    cronjob: 'cronjob.xml',
+    eventListener: 'eventListener.xml',
     file: 'files.tar',
     language: 'language/*.xml',
-    menu: null,
-    menuitem: null,
-    objectType: null,
+    menu: 'menu.xml',
+    menuitem: 'menuitem.xml',
+    objectType: 'objectType.xml',
     objectTypeDefinition: null,
```

Comment:
No, i missed it ;)
Style and script seams to be the only pips without an default name.